### ISSUE TITLE
where does the accuracy come from?

### ISSUE THREAD

**Issue by 978326187** [uzh-rpg/rpg_asynet#17]
i trained Ncaltech101 with 1500 epochs and the accuracy is **0.72** in validation dataset, **0.70** in test dataset.
Which dataset are the results in the table obtained from?
i want to know how can i get the accuracy 0.745 in **Table 1** ?

**Comment by MessikommerNico** [comment]
We used the N-Caltech101 dataset as specified in this repo.
Did the training converge or was the validation score still increasing?

**Comment by 978326187** [comment]
![image](https://user-images.githubusercontent.com/29019279/98918771-fe384a00-2508-11eb-8ff1-1c3031e93d35.png)
this is the tensorboad curve i got.
so the result in **Table 1** comes from the validation dataset of N-caltech101?

**Comment by MessikommerNico** [comment]
The reported accuracy comes from the test dataset.
We selected the checkpoints based on the best validation score, which was around 0.76

**Comment by 978326187** [comment]
the batch_size  i use  is 30

**Comment by MessikommerNico** [comment]
We used a batch size of 15.
I would rather increase the number of epochs. The validation should score should be around 0.75-0.76 at the end of the training.

**Comment by 978326187** [comment]
ok, I'll try it, thanks a lot.

**Comment by 978326187** [comment]
i have trained the model using the bath size of 15, but the result is 0.73. 
![image](https://user-images.githubusercontent.com/29019279/99742681-fe3ed800-2b0e-11eb-9f01-44d916a58c78.png)
 what should i do to get the result in paper ?

**Comment by MessikommerNico** [comment]
Seems strange your validation score is below 75% .
Maybe you could try extend the number of epochs with a Learning rate of 1e-5. So just continue the learning above with a learning rate of 1e-5.
If you want, I can also send you our trained checkpoints.

**Comment by 978326187** [comment]
thanks for your advice. If it is convenient, could you send the checkpoints  to my email address: 978326187@qq.com.

**Comment by 978326187** [comment]
could you tell me some details when training dense_vgg?

**Comment by MessikommerNico** [comment]
The training of the dense_vgg is equivalent to the training of the sparse_vgg.
